utils/httpUtils.ts

```diff
@@ -64,6 +65,9 @@ export const postWithCSRF = async (url: string, body: any, args: any, context: a
             body: JSON.stringify(body),
           });
         checkForLogin(response?.url);
+        if (fullResponse) {
```

Comment:
the default value of the parameter is a string.  I think that will technically evaluate to true, but should we make it a boolean instead?

Reply:
I had intended that the type of the parameter is a specific string "fullResponse". If you pass anything other than that specific string you will get a type error. It is easier to understand what is happening when you call the function when you use a full string. But you are right I should check for the exact string here. 

In the end, however I did not use it. I can take it away.